<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_of_task_parameters</class>
 <widget class="QWidget" name="form_of_task_parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_weight_ZZP">
     <property name="font">
      <font>
       <family>Nunito SemiBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Максимальный угол поворота, град.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вес запрета превышения допустимого угла поворота (-5 ... 5)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_max_angle">
     <property name="font">
      <font>
       <family>Nunito SemiBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_to_default">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>По умолчанию</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество ПППМ</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_save_n_quit">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Принять изменения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_sector_angle">
     <property name="font">
      <font>
       <family>Nunito SemiBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_PPPM">
     <property name="font">
      <font>
       <family>Nunito SemiBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_weight_angle_broke">
     <property name="font">
      <font>
       <family>Nunito SemiBold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Угол раствора сектора подхода, град.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вес запрета прохода через ЗЗП (-5 ... 5)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="font">
      <font>
       <family>Nunito</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
